serializers/api/nic.rb

```diff
@@ -7,7 +7,7 @@ def self.base(nic)
       name: nic.name,
       private_ipv4: nic.private_ipv4.network.to_s,
       private_ipv6: nic.private_ipv6.nth(2).to_s,
-      subnet_name: nic.private_subnet.name
+      vm_name: nic.vm&.name
```

Comment:
If we serialize the list of NICs at somewhere, it might good to load vm eagerly.